### PR TITLE
Update poemTutorial

### PR DESCRIPTION
**Pull request by conwayhub**
Take that, lonely jerk.

### CODE REVIEW

**Comment by conwayhub** [comment]
wow what a compelling change you've made, self.  good job!

File: poemTutorial [hunk 1]
```diff
@@ -5,5 +5,9 @@ A troll who's company was droll
 "TUT TUT", said the lonely soul
 "What is friendship to a troll?
 I still have standards, don't I?"
+"Standards, sure" Said the troll,
+"But the world is going to change
+around you, and you'll be left behind
+you anti-troll bigot."
 
-And the lonely soul sashayed away, as it emerged that they were a huge jerk.
+And the lonely soul felt very attacked.
```
Comment (conwayhub) [inline poemTutorial:13]: I feel very attacked right now.

File: poemTutorial [hunk 2]
```diff
@@ -5,5 +5,9 @@ A troll who's company was droll
 "TUT TUT", said the lonely soul
 "What is friendship to a troll?
 I still have standards, don't I?"
+"Standards, sure" Said the troll,
+"But the world is going to change
+around you, and you'll be left behind
+you anti-troll bigot."
```
Comment (conwayhub) [inline poemTutorial:11]: Wow, so political!

File: poemTutorial [hunk 3]
```diff
@@ -5,5 +5,9 @@ A troll who's company was droll
 "TUT TUT", said the lonely soul
 "What is friendship to a troll?
 I still have standards, don't I?"
+"Standards, sure" Said the troll,
+"But the world is going to change
+around you, and you'll be left behind
```
Comment (conwayhub) [inline poemTutorial:10]: How do you correctly punctuate a poem?  I should ask someone who secretly loves beat poetry for their insights...

Comment (wemmm) [reply poemTutorial:10]:
``` 
In another body (elsewhere)
The structure and matrices
 			seemed important
APPROPRIATE CADENCE
APPROPRIATE CAPITALISATION,
			 APPROPRIATED

How do you correctly climb a tree?
```

**Pull request merged**